include/rocksdb/convenience.h

```diff
@@ -10,12 +10,61 @@
 #include <vector>
 
 #include "rocksdb/db.h"
-#include "rocksdb/options.h"
+#include "rocksdb/status.h"
 #include "rocksdb/table.h"
 
 namespace ROCKSDB_NAMESPACE {
```

Comment:
Since it is a big change for the public interfaces, a detailed explanation is needed in the HISTORY.md